src/GraphQL/Value/ToValue.hs

```diff
@@ -21,6 +21,12 @@ instance ToValue (Value' ConstScalar) where
 instance ToValue a => ToValue [a] where
   toValue = toValue . List' . map toValue
 
+-- TODO - tom still thinks that using Maybe for nullable is maybe not
+-- the best idea.
```

Comment:
I'm starting to agree with you on this. Also think that having `Maybe a` be `Defaultable` is not a great idea.

Reasoning: the GraphQL spec says value look-up goes:
- provided value, if not there then
- default value, if not there then
- null, if nullable, otherwise
- error

Nullability should be held as a strongly distinct concept from defaultability.